adyenofficial.php

```diff
@@ -215,6 +212,17 @@ public static function getAdyenConfigNames()
             'ADYEN_ENABLE_STORED_PAYMENT_METHODS',
             'ADYEN_PAYMENT_DISPLAY_COLLAPSE'
         );
+
+        if ($mode === 'test') {
```

Comment:
I think this check needs to be done [here](https://github.com/Adyen/adyen-prestashop/pull/203/files#diff-6fb035a8c069d2f2a8569fffb0e36036621d864e8e76d6ef65bb714253a51615R851) when validating the input fields